src/app/pages/LoginPage/components/LoginModalWindow1.tsx

```diff
@@ -159,11 +154,11 @@ const styles = {
     w-[300px]
     h-[40px]
     my-[12px]
-    bg-black dark:bg-white
+    bg-primaryBg-dark dark:bg-primaryBg-light
```

Comment:
Seems strange that for light theme used color from dark theme and vice versa

Reply:
Well, it is logical that there will be a dark button on a light background, and vice versa on a dark background - a white button.

Reply:
If color from dark theme it don't means that all pallete do not have bright colors. As for backgrounds it's more likely to be true, and this is good enough